homeassistant/components/unifiprotect/strings.sensor.json

```diff
@@ -0,0 +1,7 @@
+{
+    "state": {
+        "unifiprotect__license_plate": {
+            "none": "Clear"
+        }
```

Comment:
I'm missing a translation for `DEVICE_CLASS_DETECTION`.

Reply:
That one is deprecated and will be removed in a future release.

Reply:
How old is it, has it already been released?

Reply:
Yes.

Reply:
Ok.